modules/services/swayosd.nix

```
@@ -57,6 +66,7 @@ in {
           Type = "simple";
           ExecStart = "${cfg.package}/bin/swayosd-server"
             + (optionalString (cfg.display != null) " --display ${cfg.display}")
+            + (optionalString (cfg.style != null) " --style \"${cfg.style}\"")
```

Should be more robust to use `escapeShellArg`:
```suggestion
            + (optionalString (cfg.style != null) " --style ${escapeShellArg cfg.style}")
```